src/extension/mapml/src/main/java/org/geoserver/mapml/tcrs/TiledCRSParams.java

```diff
@@ -30,6 +34,13 @@ public TiledCRSParams(
         this.TILE_SIZE = tileSize;
         this.origin = origin;
         this.scales = scales;
+        // although they are redundant since one is the inverse of the other
+        // we store both of them for quick access (e.g. debug) without need
```

Comment:
Looks like they are being used for more than just debug?

Reply:
Correct. they were not needed during the first phases of the development. Removing that comment.